### PR TITLE
2.10.20 Release Notes

### PR DESCRIPTION
**Pull request by billpalombi**
2.10.20 Release Notes

### CODE REVIEW

File: RELEASE-NOTES.md [hunk 1]
```diff
@@ -1,5 +1,26 @@
...
+### Prefect no longer supports Python 3.7
+
+Python 3.7 reached end-of-life on 27 Jun 2023. Consistent with our warning, this release drops Python 3.7 support. Prefect now requires Puython 3.8 or later.
+
+### Enhancements
+- Add UUID validation for webhook, so they fail erlier and more clearly — https://github.com/PrefectHQ/prefect/pull/10005
+- Clarify Dockerfile rename language in `prefect deploy` CLI prompt — https://github.com/PrefectHQ/prefect/pull/10124
+- Improve `prefect deploy` error message — https://github.com/PrefectHQ/prefect/pull/10175
+- Add `work_pool_name` to `Deployment` docstring — https://github.com/PrefectHQ/prefect/pull/10174
+
```
Comment (pleek91) [inline RELEASE-NOTES.md:18]:
```suggestion

### Fixes
- Unable to select correct date/time for deployment flow run parameters — https://github.com/PrefectHQ/prefect-ui-library/pull/1554
- List[Literal] input selection not working as expected — https://github.com/PrefectHQ/prefect-ui-library/pull/1556
- JSON fields incorrectly escaping values — https://github.com/PrefectHQ/prefect-ui-library/pull/1557

```

File: RELEASE-NOTES.md [hunk 2]
```diff
@@ -1,5 +1,26 @@
...
+### Resolving UI form input issues
+
+This off-cycle release resolves bugs preventing UI form inputs from being rendered and parsed correctly. 
+
+### Prefect no longer supports Python 3.7
+
+Python 3.7 reached end-of-life on 27 Jun 2023. Consistent with our warning, this release drops Python 3.7 support. Prefect now requires Puython 3.8 or later.
+
+### Enhancements
+- Add UUID validation for webhook, so they fail erlier and more clearly — https://github.com/PrefectHQ/prefect/pull/10005
```
Comment (billpalombi) [inline RELEASE-NOTES.md:14]:
```suggestion
- Add UUID validation for webhook CLI commands, so they fail earlier and more clearly — https://github.com/PrefectHQ/prefect/pull/10005
```

**Review by pleek91 — APPROVED** [review]

File: RELEASE-NOTES.md [hunk 3]
```diff
@@ -1,5 +1,31 @@
...
+
+### Fixes
+- Unable to select correct date/time for deployment flow run parameters — https://github.com/PrefectHQ/prefect-ui-library/pull/1554
+- List[Literal] input selection not working as expected — https://github.com/PrefectHQ/prefect-ui-library/pull/1556
+- JSON fields incorrectly escaping values — https://github.com/PrefectHQ/prefect-ui-library/pull/1557
+
+### Contributors
+- @toby-coleman
+
+**All changes**: https://github.com/PrefectHQ/prefect/compare/2.10.19...preview
```
Comment (serinamarie) [inline RELEASE-NOTES.md:27]:
```suggestion
**All changes**: https://github.com/PrefectHQ/prefect/compare/2.10.19...2.10.20
```

File: RELEASE-NOTES.md [hunk 4]
```diff
@@ -1,5 +1,29 @@
 # Prefect Release Notes
 
+## Release 2.10.20
+
+### Resolving UI form input issues
+
+This off-cycle release resolves bugs preventing UI form inputs from being rendered and parsed correctly, including:
+- Dates & times — https://github.com/PrefectHQ/prefect-ui-library/pull/1554
+- List value selections — https://github.com/PrefectHQ/prefect-ui-library/pull/1556
```
Comment (billpalombi) [inline RELEASE-NOTES.md:9]:
```suggestion
- List values — https://github.com/PrefectHQ/prefect-ui-library/pull/1556
```

File: RELEASE-NOTES.md [hunk 5]
```diff
@@ -1,5 +1,29 @@
...
+- Dates & times — https://github.com/PrefectHQ/prefect-ui-library/pull/1554
+- List values — https://github.com/PrefectHQ/prefect-ui-library/pull/1556
+- JSON fields — https://github.com/PrefectHQ/prefect-ui-library/pull/1557
+
+### Prefect no longer supports Python 3.7
+
+Python 3.7 reached end-of-life on 27 Jun 2023. Consistent with our warning, this release drops Python 3.7 support. Prefect now requires Puython 3.8 or later.
+
+### Enhancements
+- Add UUID validation for webhook CLI commands, so they fail earlier and more clearly — https://github.com/PrefectHQ/prefect/pull/10005
```
Comment (serinamarie) [inline RELEASE-NOTES.md:17]:
```suggestion
- Add UUID validation for webhook CLI commands to raise errors earlier and more clearly — https://github.com/PrefectHQ/prefect/pull/10005
```

File: RELEASE-NOTES.md [hunk 6]
```diff
@@ -1,5 +1,29 @@
 # Prefect Release Notes
 
+## Release 2.10.20
+
+### Resolving UI form input issues
+
+This off-cycle release resolves bugs preventing UI form inputs from being rendered and parsed correctly, including:
```
Comment (discdiver) [inline RELEASE-NOTES.md:7]:
```suggestion
This release resolves bugs preventing UI form inputs from being rendered and parsed correctly, including:
```

File: RELEASE-NOTES.md [hunk 7]
```diff
@@ -1,5 +1,29 @@
...
+- List values — https://github.com/PrefectHQ/prefect-ui-library/pull/1556
+- JSON fields — https://github.com/PrefectHQ/prefect-ui-library/pull/1557
+
+### Prefect no longer supports Python 3.7
+
+Python 3.7 reached end-of-life on 27 Jun 2023. Consistent with our warning, this release drops Python 3.7 support. Prefect now requires Puython 3.8 or later.
+
+### Enhancements
+- Add UUID validation for webhook CLI commands to raise errors earlier and more clearly — https://github.com/PrefectHQ/prefect/pull/10005
+- Clarify Dockerfile rename language in `prefect deploy` CLI prompt — https://github.com/PrefectHQ/prefect/pull/10124
```
Comment (serinamarie) [inline RELEASE-NOTES.md:18]:
```suggestion
- Clarify Dockerfile rename prompt in `prefect deploy` — https://github.com/PrefectHQ/prefect/pull/10124
```
just a suggestion.. not totally sure if it's improved

Comment (billpalombi) [reply RELEASE-NOTES.md:18]: Yours is clearer

**Review by serinamarie — APPROVED** [review]
LGTM!

**Review by discdiver — COMMENTED** [review]
As we occasionally have releases on different days, just a minor wording change.

**Comment by zanieb** [comment]
@billpalombi minor note, the versioning scheme says that deprecated features will be dropped in minor version bumps. I'd avoid doing so in patch versions if feasible :)

**Pull request merged**